### PR TITLE
Awakened King DLC Items

### PR DESCRIPTION
**Pull request by aescolastico**
Apologies if this already being worked on. Tried getting the ball rolling on new DLC.

That being said, Im clearly missing something or Im defining the objects incorrectly in the game.json file. The save files are riddled with binary noise so its hard to read them. 
 
Either way, the items I added are not appearing as missing.

### CODE REVIEW

File: RemnantSaveGuardian/game.json [hunk 1]
```diff
@@ -510,6 +512,54 @@
...
       ]
     },
+    "DLC1": {
+      "Ring_WoodRing": [
+        {
+          "name": "/Items/Trinkets/Rings/WoodRing/Ring_WoodRing",
+          "notes": "Can be found as a random drop in Losomn."
+        }
+      ],
+      "Weapon_AbyssalHook": {
```
Comment (dyfrgi) [inline RemnantSaveGuardian/game.json:522]: Wrong type here - this should be an array of objects, like you did for Wood Ring.

**Review by dyfrgi — CHANGES_REQUESTED** [review]
Thanks for writing this! I think that there are some more items known now as well, take a look at https://www.reddit.com/r/remnantgame/comments/17w04g7/awakened_king_dlc_item_list_as_of_now/.

I figured out why it's not working, it's due to JSON types. If you look at the in-program log when running it you can see the error message, though unfortunately it doesn't include the context of where it is in the JSON.

**Comment by MrWastedPanic** [comment]
I have been writing down the items I have come across. Once this is committed I can pull and add them or if you want, I can send them to you some other way.

File: RemnantSaveGuardian/game.json [hunk 2]
```diff
@@ -888,8 +938,8 @@
         },
         {
           "name": "/Items/Armor/Misc/Armor_Head_RedPrince",
-          "notes": "kill with Assassins Dagger",
-          "armorSet":  false
+          "notes": "Rewarded by killing the Red Prince with the 'Assassin's Dagger' melee weapon.",
```
[the remark this answers is not in the document]
Comment (aescolastico) [reply RemnantSaveGuardian/game.json:941]: agreed

File: RemnantSaveGuardian/game.json [hunk 3]
```diff
@@ -446,7 +448,7 @@
         {
           "name": "/Items/Armor/Armor_Head_LodestoneCrown",
           "notes": "On the boat inside Kolkets locked room",
-          "armorSet":  false
+          "armorSet": false
```
[the remark this answers is not in the document]
Comment (aescolastico) [reply RemnantSaveGuardian/game.json:451]: agreed

**Comment by aescolastico** [comment]
> Thanks for writing this! I think that there are some more items known now as well, take a look at https://www.reddit.com/r/remnantgame/comments/17w04g7/awakened_king_dlc_item_list_as_of_now/.

while this list is awesome. we kind of need the gamestrings otherwise wed be guessing at what the devs named these items in the game files. which is sometimes is easy to do if you have the in-game names, but id prefer to be confident about what were proposing for this project. if someone with all these items provides their profile.sav, we could extract these strings.

**Comment by MrWastedPanic** [comment]
> while this list is awesome. we kind of need the gamestrings otherwise wed be guessing at what the devs named these items in the game files. which is sometimes is easy to do if you have the in-game names, but id prefer to be confident about what were proposing for this project. if someone with all these items provides their profile.sav, we could extract these strings.

I have a large number of these items in my inventory if you want my profile.sav let me know

**Comment by aescolastico** [comment]
> > while this list is awesome. we kind of need the gamestrings otherwise wed be guessing at what the devs named these items in the game files. which is sometimes is easy to do if you have the in-game names, but id prefer to be confident about what were proposing for this project. if someone with all these items provides their profile.sav, we could extract these strings.
> 
> I have a large number of these items in my inventory if you want my profile.sav let me know

yes that would be helpful

**Comment by MrWastedPanic** [comment]
> > > while this list is awesome. we kind of need the gamestrings otherwise wed be guessing at what the devs named these items in the game files. which is sometimes is easy to do if you have the in-game names, but id prefer to be confident about what were proposing for this project. if someone with all these items provides their profile.sav, we could extract these strings.
> > 
> > 
> > I have a large number of these items in my inventory if you want my profile.sav let me know
> 
> yes that would be helpful

Let me know how I can send it to you and it's yours

**Comment by aescolastico** [comment]
@MrWastedPanic you can just attach the file to a comment in this very thread.

**Comment by MrWastedPanic** [comment]
> @MrWastedPanic you can just attach the file to a comment in this very thread.

Let me know if that worked for you

[profile.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13386357/profile.zip)

File: RemnantSaveGuardian/game.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (MrWastedPanic) [reply RemnantSaveGuardian/game.json:451]: Maybe revert the whitespace changes till after the DLC piece is added (or at least the known pieces) so we can get this merged and continue working?

File: RemnantSaveGuardian/game.json [hunk 4]
```diff
@@ -1648,8 +1698,8 @@
           "notes": "Purchased from Reggie at Ward 13 after defeating all bosses at least once."
         },
         {
-            "name": "/Items/Trinkets/Rings/BrightSteelRing/Ring_BrightSteelRing",
-            "notes": "Purchased from Reggie at Ward 13 after completing 15 biomes on any difficulty."
+          "name": "/Items/Trinkets/Rings/BrightSteelRing/Ring_BrightSteelRing",
```
[the remark this answers is not in the document]
Comment (MrWastedPanic) [reply RemnantSaveGuardian/game.json:1721]: Again, maybe wrap this change to the whitespace changes mentioned above and call it a QoL update after the DLC information?

**Comment by aescolastico** [comment]
I'll have to check, but i think PR#193 covers all these changes and more, so i can probably just close this PR.

**Comment by Razzmatazzz** [comment]
@aescolastico there were three competing PRs for the DLC items, but two of them (including this one) were marked as drafts so I merged the one that was marked as ready. For future PRs, be sure to mark them as ready when they're ready so I know it's time to review and I can get them merged.

**Pull request closed without merging**